src/Apps/Legion/Contexts/WizardDataContext/WizardDataContext.tsx

```diff
@@ -121,6 +122,21 @@ export const WizardDataContextReducer: (
                 deletePropertyById(propertyId, draft);
                 break;
             }
+            case WizardDataContextActionType.ADD_COOKED_SOURCE_ASSETS: {
+                const { data } = action.payload;
+                draft.properties = draft.properties.concat(data.properties);
+                const types = deepCopy(data.types);
+                draft.types = draft.types.concat(
+                    types.map((t, idx) => {
+                        t.color = getHighChartColorByIdx(
```

Comment:
We should make this a utils function for Legion, no longer rely on highchart colors

Reply:
agreed. Especially since it's not for a chart.

Reply:
I'm ok with the palette, it just felt weird using a chart utility function for this.